<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加待办</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icon.qrc">
    <normaloff>:/images/Icon_min.png</normaloff>:/images/Icon_min.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_eventType">
     <item>
      <widget class="QLabel" name="label_eventType">
       <property name="text">
        <string>事件类型</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_eventType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_eventTypeOnce">
          <property name="text">
           <string>一次性事件</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_eventTypeLoop">
          <property name="text">
           <string>循环事件</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_loopType">
     <item>
      <widget class="QLabel" name="label_loopType">
       <property name="text">
        <string>循环周期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_loopType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>SimSun</family>
        </font>
       </property>
       <property name="currentIndex">
        <number>5</number>
       </property>
       <item>
        <property name="text">
         <string>每年</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每月</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每周</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每天</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>每小时</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_date">
     <item>
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>日期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_weekday">
     <item>
      <widget class="QLabel" name="label_weekday">
       <property name="text">
        <string>星期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_weekday">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Symbol</family>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>不匹配</string>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <property name="maxVisibleItems">
        <number>8</number>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>星期日</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期一</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期二</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期三</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期四</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期五</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>星期六</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>不匹配</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_matchTime">
     <item>
      <widget class="QLabel" name="label_matchTime">
       <property name="text">
        <string>匹配时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_matchTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_matchTimeYes">
          <property name="text">
           <string>是</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_matchTimeNo">
          <property name="text">
           <string>否</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_time">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_time">
         <property name="text">
          <string>时间</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_event">
     <item>
      <widget class="QLabel" name="label_event">
       <property name="text">
        <string>事件</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_event"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_eventDescription">
     <item>
      <widget class="QLabel" name="label_eventDescription">
       <property name="text">
        <string>描述</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_eventDescription">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_displayType">
     <item>
      <widget class="QLabel" name="label_displayType">
       <property name="text">
        <string>显示类型</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_displayType">
       <item>
        <property name="text">
         <string>文本</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>图片</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>音频</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>视频</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>网站</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_mediaFile">
       <property name="text">
        <string>资源路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_mediaFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_selectFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_button">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>401</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_pushbutton">
       <item>
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>完成</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/icon.qrc"/>
 </resources>
 <connections/>
</ui>
